libraries/thingy_52/m_ui.c

```diff
@@ -37,65 +37,48 @@
  */
 
 #include "m_ui.h"
+#include "app_util_platform.h"
 #include "drv_ext_light.h"
-#include "app_error.h"
-#include <stdlib.h>
-#include <string.h>
-#include "pca20020.h"
 #include "nrf_drv_gpiote.h"
-#include "app_scheduler.h"
-#include "app_util_platform.h"
+#include "pca20020.h"
+#include <stdint.h>
+#include <stdlib.h>
 
-#define  NRF_LOG_MODULE_NAME "m_ui          "
+#define NRF_LOG_MODULE_NAME "m_ui"
 #include "nrf_log.h"

```

Comment:
```
#define NRF_LOG_MODULE_NAME "m_ui"
#include "nrf_log.h"
```
We can also delete these 2 lines as we are not using them

Reply:
is deleted